docs/sources/alerting/alerting-rules/create-mimir-loki-managed-recording-rule.md

```diff
@@ -96,18 +96,22 @@ Select your data source and enter a query.
 
 ## Configure Grafana-managed recording rules
 
-To configure Grafana-managed recording rules, complete the following steps.
+Configure Grafana-managed recording rules.
+
+{{< admonition type="note" >}}
+This feature is only available for Grafana OSS and Enterprise users. It is not available in Grafana Cloud.
+{{< /admonition >}}
 
 ### Before you begin
 
-If you are using Grafana OSS, enable the `grafanaManagedRecordingRules` feature flag.
+- Enable the `grafanaManagedRecordingRules` [feature flag](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/setup-grafana/configure-grafana/feature-toggles/).
 
-### Steps
+To configure Grafana-managed recording rules, complete the following steps.
 
 1. Click **Alerts & IRM** -> **Alerting** ->
    **Alert rules**.
-1. Select **Rule type** -> **Recording**.
-1. Scroll to the **Grafana-managed section** and click **+New recording rule**.
+2. Select **Rule type** -> **Recording**.
```

Comment:
this step is not necessary, right? i mean, you can create a recording rule without this filter

Reply:
yes - thanks - missed that!!!